package.json

```diff
@@ -63,6 +63,10 @@
 								"type": "string",
 								"description": "SPARQL endpoint host"
 							},
+							"path": {
+								"type": "string",
+								"description": "Path component of SPARQL endpoint, eg '/sparql' "
```

Comment:
It would be great if this configuration item (if not supplied) defaulted to `/sparql`

Reply:
I tried adding the default value in package.json but it doesn't have any effect and I don't know why.  We could set the default value in code instead, however, it would seem cleaner to me if the '/sparql' path was not added by default and we just explicitly include the path with the host (and port) as a single config property (called something like endpoint) and then there is no need to have a separate path config property.  I also don't see any reason to have a separate config property for the scheme (you call it protocol, ie http/https) and it could be combined with the host, port and path to make a single URI property which would provide the most flexibility.

Reply:
Sorry for the late response, yes I was worried about backwards compatibility. Your idea to add a single new (optional) 'endpoint_url' configuration sounds like the best approach.

Reply:
Hi,

 I'm interested in this feature as well.

 Couldn't you just check for the absence of a configured value and use '/sparql' ?

Reply:
Or even simpler, just provide everything as one value: URL

Reply:
Interested as well. Can't use this on a custom endpoint at the moment, and I would use this extension, if possible.

Reply:
Check out my fork of the repo; I've implemented support for "path".